.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Sorry, my fault on this. I think I used a different text editor and it dumped this in… Back to Atom for me :-)